frontend/components/MainPage/header/AgentButton/AgentNotRunningButton.tsx

```diff
@@ -143,25 +147,27 @@ export const AgentNotRunningButton = () => {
       selectedAgentConfig.operatingThresholds[WalletOwnerType.Master];
     const tokenSymbol =
       CHAIN_CONFIG[selectedAgentConfig.evmHomeChainId].nativeToken.symbol;
-    const agentSafeNativeBalance = serviceSafeBalances?.find(
-      ({ symbol }) => symbol === tokenSymbol,
-    )?.balance;
```

Comment:
all funded on the agent safe is handled by middleware, frontend should only really care about master wallets, rest is automated